history-network.md

```diff
@@ -130,6 +130,17 @@ _MAX_HEADER_LENGTH = 2**13  # = 8192
 MAX_ENCODED_UNCLES_LENGTH = _MAX_HEADER_LENGTH * 2**4  # = 2**17 ~= 131k
 # Maximum number of uncles is currently 2. Using 16 leaves some room for the
 # protocol to increase the number of uncles.
+
+MAX_WITHDRAWAL_COUNT = 16
+# Number sourced from EIP-4895
+
+MAX_WITHDRAWAL_LENGTH = 64
+# Withdrawal: index (u64), validator_index (u64), address, amount (u64)
+#   - 8 + 8 + 20 + 8 = 44 bytes
+#   - round up to nearest power of 2 => 64
```

Comment:
Note that this isn't actually required for SSZ. But an increase is necessary anyhow because of rlp encoding overhead. Perhaps we should mention a reason for this increase.